db/migrate/20210421190438_add_body_markdown_tsvector_index_to_users.rb

```diff
@@ -0,0 +1,27 @@
+class AddBodyMarkdownTsvectorIndexToUsers < ActiveRecord::Migration[6.1]
```

Comment:
the index is being added to Comments and not users, probably need to update the name.

Reply:
Classic copy/pasta, great catch - updated!